router/h2/src/e2e/scala/io/buoyant/router/h2/FlowControlEndToEndTest.scala

```diff
@@ -2,11 +2,9 @@ package io.buoyant.router
 package h2
 
 import com.twitter.finagle.buoyant.h2._
-import com.twitter.io.Buf
 import com.twitter.logging.Level
-import com.twitter.util.{Future, Promise}
+import com.twitter.util.Promise
 import io.buoyant.test.FunSuite
-import scala.collection.mutable.ListBuffer
```

Comment:
Were these imports unused? This looks like an unrelated change at a glance?

Reply:
yup, an unrelated "clean-as-I-go" change